### PR TITLE
Add troubleshooting guide

### PR DESCRIPTION
**Pull request by MarcoRossignoli**
Added troubleshooting guide to take advantage of new log infrastructure.

/cc @tonerdo @petli

### CODE REVIEW

File: Documentation/Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Troubleshooting
+
+## Msbuild integration
```
Comment (tonerdo) [inline Documentation/Troubleshooting.md]: Title case

```suggestion
## Msbuild Integration
```

File: Documentation/Troubleshooting.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+dotnet test test\coverlet.core.tests\coverlet.core.tests.csproj -c debug /p:CollectCoverage=true /p:CoverletOutputFormat=opencover /p:Include=[coverlet.*]* -verbosity:diagnostic -bl:msbuild.binlog -noconsolelogger
+```
+
+2) Download http://msbuildlog.com/
+3) Open `msbuild.binlog` generated and search for '[coverlet]' logs
+ 
+![File](images/file.png)
+
+## Coverlet dotnet tool
```
Comment (tonerdo) [inline Documentation/Troubleshooting.md]: Same here:

```suggestion
## Coverlet Global Tool
```

**Comment by MarcoRossignoli** [comment]
@tonerdo we can merge this

**Comment by MarcoRossignoli** [comment]
ping @tonerdo

**Comment by tonerdo** [comment]
@MarcoRossignoli do you have merge rights?

**Comment by MarcoRossignoli** [comment]
Yes @tonerdo but we never talked about "when I can merge", if it's ok for you we could state that if you "Approve" and also to me it's ok(I approve) I can merge, what do you think?

**Comment by tonerdo** [comment]
@MarcoRossignoli I'm fine with that

**Pull request merged**